scubagoggles/reporter/reporter.py

```diff
@@ -227,45 +262,51 @@ def rego_json_to_html(test_results_data : str, product : list, out_path : str,
             tests = [test for test in test_results_data if test['PolicyId'] == control['Id']]
             if len(tests) == 0:
                 # Handle the case where Rego doesn't output anything for a given control
-                report_stats['Error'] += 1
+                report_stats['Errors'] += 1
                 issues_link = f'<a href="{SCUBA_GITHUB_URL}/issues" target="_blank">GitHub</a>'
                 table_data.append({
                     'Control ID': control['Id'],
                     'Requirement': control['Value'],
                     'Result': "Error - Test results missing",
                     'Criticality': "-",
-                    'Details': f'Report issue on {issues_link}'
-                })
+                    'Details': f'Report issue on {issues_link}'})
                 warnings.warn(f"No test results found for Control Id {control['Id']}",
                     RuntimeWarning)
             else:
                 for test in tests:
                     failed_prereqs = get_failed_prereqs(test, successful_calls, unsuccessful_calls)
                     if len(failed_prereqs) > 0:
                         result = "Error"
-                        report_stats["Error"] += 1
+                        report_stats["Errors"] += 1
                         failed_details = get_failed_details(failed_prereqs)
                         table_data.append({
                             'Control ID': control['Id'],
                             'Requirement': control['Value'],
                             'Result': "Error",
                             'Criticality': test['Criticality'],
-                            'Details': failed_details
-                        })
+                            'Details': failed_details})
                     else:
                         result = get_test_result(test['RequirementMet'], test['Criticality'],
                         test['NoSuchEvent'])
-
-                        report_stats[result] = report_stats[result] + 1
-                        details = test['ReportDetails']
+                        if result in {"No events found","N/A"}:
+                            report_stats["Manual"] += 1
+                            details = test['ReportDetails']
+                        elif result == "Warning":
+                            report_stats["Warnings"] += 1
+                            details = test['ReportDetails']
+                        elif result == "Fail":
+                            report_stats["Failures"] += 1
+                            details = test['ReportDetails']
+                        elif result == "Pass":
+                            report_stats["Passes"] += 1
+                            details = test['ReportDetails']
```

Comment:
This segment has merge conflicts due to my PR which was just merged in: https://github.com/cisagov/ScubaGoggles/pull/246.

![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/9ee3e4e3-8e9d-4e74-ad8d-0191d8f92472)

Basically, I moved the code updating the `report_stats` lower in the function to correct this bug: https://github.com/cisagov/ScubaGoggles/issues/184.

The `report_stats` are now incremented here instead:
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/8bc69257-44c8-4cdf-8ec5-40c0a678c85e)